apps/resources/models.py

```diff
@@ -0,0 +1,67 @@
+from django.conf import settings
+from django.db import models
+
+from ckeditor.fields import RichTextField
+from sorl.thumbnail import ImageField
+
+from directory.models import Organisation
+from tags.models import Tag
+
+from .managers import ResourceManager
+
+
+class Resource(models.Model):
+    title = models.CharField(max_length=256, unique=True)
+    abstract = models.TextField(
+        max_length=140,
+        help_text='This text will appear in search results',
+    )
+    content = RichTextField()
+    tags = models.ManyToManyField(
+        Tag,
+        limit_choices_to=models.Q(level=1) | models.Q(level=2),
```

Comment:
Slightly unsure on this limit.

Tags could be moved around due to MPTT anyway, which could get around this restriction.

Maybe go ahead and see what happens later?